frame/support/test/tests/pallet.rs

```diff
@@ -839,15 +837,54 @@ fn pallet_hooks_expand() {
 #[test]
 fn pallet_on_genesis() {
 	TestExternalities::default().execute_with(|| {
-		assert_eq!(pallet::Pallet::<Runtime>::storage_version(), None);
+		assert_eq!(pallet::Pallet::<Runtime>::active_storage_version(), StorageVersion::new(0));
 		pallet::Pallet::<Runtime>::on_genesis();
 		assert_eq!(
-			pallet::Pallet::<Runtime>::storage_version(),
-			Some(pallet::Pallet::<Runtime>::current_version()),
+			pallet::Pallet::<Runtime>::current_storage_version(),
+			pallet::Pallet::<Runtime>::active_storage_version(),
```

Comment:
A personal nit pick, I think these two names can be too easily confused with one another.

I would want to name one `on_chain_storage_version`

Reply:
I'm fine :D  I'm bad at naming :P 

So, active to `on_chain_storage_version`?

Reply:
yup!